jose/backends/__init__.py

```diff
@@ -10,19 +10,9 @@
     from jose.backends.cryptography_backend import CryptographyRSAKey as RSAKey  # noqa: F401
 except ImportError:
     try:
-        from jose.backends.pycrypto_backend import RSAKey  # noqa: F401
-
-        # time.clock was deprecated in python 3.3 in favor of time.perf_counter
-        # and removed in python 3.8. pycrypto was never updated for this. If
-        # time has no clock attribute, let it use perf_counter instead to work
-        # in 3.8+
-        # noinspection PyUnresolvedReferences
-        import time
-        if not hasattr(time, "clock"):
-            time.clock = time.perf_counter
-
-    except ImportError:
         from jose.backends.rsa_backend import RSAKey  # noqa: F401
+    except ImportError:
+        RSAKey = None
```

Comment:
This is new behavior. I assumed this library needs to support this (w/ reduced functionality, obviously)